packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/index.js

```diff
@@ -73,11 +76,22 @@ const TransferTokenCreateView = () => {
     },
     {
       enabled: !isCreating && !transferToken,
-      onError() {
-        toggleNotification({
-          type: 'warning',
-          message: { id: 'notification.error', defaultMessage: 'An error occured' },
-        });
+      onError(err) {
+        if (err?.response?.data?.error?.details?.code === 'INVALID_TOKEN_SALT') {
```

Comment:
I think in the case of an error you always know that `err`, `response` and `data` exist. This would simplify it a bit:

```suggestion
        if (err.response.data.error.details?.code === 'INVALID_TOKEN_SALT') {
```

Reply:
done